htdocs/AjaxHelper.php

```diff
@@ -80,6 +78,13 @@
     header("HTTP/1.1 400 Bad Request");
     exit(5);
 }
+
+$m = Module::factory($Module);
+if ($anonymous === true && $m->isPublicModule() !== true) {
```

Comment:
Isn't it cleaner to do `$m->isPublicModule() === false`? ESLint taught me to avoid unnecessary negation